data/presentations.yaml

```
@@ -393,4 +393,129 @@
+  category: case-study
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  description: Paul gives a brief description on how Parsec can make use of SPIFFE identities to support multitenancy.
+- title: "SPIFFE and SPIRE: Architecture Deep Dive"
+  link: https://youtu.be/ZJxq8hPgYVI
+  youtube: ZJxq8hPgYVI
+  presenters: Andrew Harding and Evan Gilman
+  organizations: VMware, Scytale
+  category: overview
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  description: Andrew and Evan offer a complete explanation about SPIFFE and SPIRE architecture, including main topics as node and workload attestation, key management and rotation strategy.
+- title: "Securing Kafka with SPIFFE"
+  link: https://youtu.be/4pfY0uFW7yk
+  youtube: 4pfY0uFW7yk
+  presenters: Jonathan Oddy and Levani Kokhreidze
+  organizations: TransferWise
+  category: case-study
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  description: Jonathan and Levani explain how SPIFFE and Envoy is used to secure Kafka's client-broker communication, by solving common problems as certificate distribution at large scale.
+- title: "Using SPIRE in Production at Uber"
+  link: https://youtu.be/o98SFmW4akQ
+  youtube: o98SFmW4akQ
+  presenters: Andrew Moore
+  organizations: Uber
+  category: case-study
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  description: This session provides an overview of how Uber uses SPIFFE and SPIRE for workload authentication in a diverse deployment environment.
+- title: "SPIFFE at GitHub"
+  link: https://youtu.be/vX8SS5wQuY8
+  youtube: vX8SS5wQuY8
+  presenters: Eric Lee
+  organizations: GitHub
+  category: case-study
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  description: Eric explains how SPIRE is used in GitHub giving implementation details about agent operation and custom node selectors, making use of the platform-agnostic characteristic of SPIRE.
+- title: "Passport App: The role of SPIFFE and SPIRE in a return to work solution"
+  link: https://youtu.be/nZXCbiS6wfE
+  youtube: nZXCbiS6wfE
+  presenters: Frederick Kautz
+  organizations: doc.ai
+  category: case-study
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  descripion: Frederick demonstrates a SPIFFE/SPIRE enabled solution which will help employers manage there return to work strategy, focusing on privacy.
+- title: "10 Lessons From Migrating to SPIFFE After 10 Years Of Service Identity at Square"
+  link: https://youtu.be/x642wq7lbpY
+  youtube: x642wq7lbpY
+  presenters: Mat Byczkowski
+  organizations: Square
+  category: case-study
+  date: 17-Nov-2020
+  venue: SPIFFE + SPIRE at KubeCon + CloudNativeCon NA 2020
+  description: Mat exposes the principal benefits of migrating from a homegrown service identity issuer to SPIFFE, showing what the team learned in this process.
```

"Exposes" isn't quite right here. It can mean to reveal something bad, like "expose a lie."

```suggestion
  description: Mat describes the principal benefits of migrating from a homegrown service identity issuer to SPIFFE, showing what the team learned in this process.
```

---

good thing to know!